solutions/devsprint-riccardo-1/OnboardingChallenge.xcodeproj/project.pbxproj

```diff
@@ -457,7 +450,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = B8F644M47X;
+				DEVELOPMENT_TEAM = WA9JUKEPEP;
```

Comment:
could you revert the change done to the development team? 🙏 Otherwise everyone will get the wrong team in the project.